python/tests/expressions/test_literals.py

```diff
@@ -398,8 +397,8 @@ def test_fixed_literal():
 
 
 def test_binary_literal():
-    bin_lit012 = literal(bytearray([0x00, 0x01, 0x02]))
-    bin_lit013 = literal(bytearray([0x00, 0x01, 0x03]))
+    bin_lit012 = literal(bytes([0x00, 0x01, 0x02]))
+    bin_lit013 = literal(bytes([0x00, 0x01, 0x03]))
```

Comment:
Is this change still needed? I thought that `literal(b: bytearray)` was able to produce `Literal[bytes]`. If that's not the case then this is fine.

Reply:
I was able to achieve this using the `@overload` method, but that had other issues downstream. Personally I've always been using `bytes`, and you can easily cast a `bytearray` into `bytes`, so I think we're good here.

Reply:
Sounds good to me.